recipes/ylearn/meta.yaml

```diff
@@ -6,17 +6,17 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/DataCanvasIO/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: c54fa3e18ad6dce9e7ac0997db809572e7ebe2cf22be502aabcc84c3282786f6
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eca57cff71847db16c555fe596fbf84da598c4f59f1515db9136d8554c35c55e
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - cython
```

Comment:
I did not check upstream's setup.py but re-cythonizing should be an option.
I'll leave that to you in the feedstock. It is preferred b/c it makes migrations to new python versions easier.

Reply:
Thanks! 
To support re-cythonizing, the setup settings need to be updated.  I'll  fix it  at next release.